File: Java-Core/src/main/java/presto/declaration/AnyNativeCategoryDeclaration.java
package presto.declaration;

import presto.grammar.Identifier;
import presto.grammar.MethodDeclarationList;
import presto.grammar.NativeAttributeBindingListMap;
import presto.grammar.NativeCategoryBindingList;
import presto.utils.IdentifierList;

public class AnyNativeCategoryDeclaration extends NativeCategoryDeclaration {
	
	static private AnyNativeCategoryDeclaration instance = new AnyNativeCategoryDeclaration();
	
	static public AnyNativeCategoryDeclaration getInstance() {
		return instance;
	}
	
	private AnyNativeCategoryDeclaration() {
		super(new Identifier("Any"), new IdentifierList(), 
				new NativeCategoryBindingList(), 
				new NativeAttributeBindingListMap(),
				new MethodDeclarationList());
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/declaration/AnyNativeCategoryDeclaration.js
var NativeCategoryDeclaration = require("./NativeCategoryDeclaration").NativeCategoryDeclaration;

function AnyNativeCategoryDeclaration() {
    NativeCategoryDeclaration.call(this, "Any", [], [], [], []);
    return this;
}

AnyNativeCategoryDeclaration.prototype = Object.create(NativeCategoryDeclaration.prototype);
AnyNativeCategoryDeclaration.prototype.constructor = AnyNativeCategoryDeclaration;

AnyNativeCategoryDeclaration.instance = new AnyNativeCategoryDeclaration();

exports.AnyNativeCategoryDeclaration = AnyNativeCategoryDeclaration;
